styles/app.css

```diff
@@ -0,0 +1,132 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.navbar {
+  background: #2CC5E1;
+  overflow: auto;
+  margin: auto;
+}
+
+.navbar-left {
+ float: left;
+ color: white;
+ line-height: 2rem;
+ padding: 1rem;
```

Comment:
Try removing the left and right padding on the left and right navbar elements so that they line up with the grid below.